internal/server/server.go

```diff
@@ -32,12 +32,12 @@ func Start(ctx context.Context, logger *slog.Logger, serverConf config.Server, s
 		gin.SetMode(gin.ReleaseMode)
 	}
 
-	r := setupRouter(ctx, logger, serverConf, s)
+	r := setupRouter(ctx, logger, serverConf, s, dockerPingFunc)
 
-	var server *http.Server
-	server = &http.Server{
-		Addr:    fmt.Sprintf(":%d", serverConf.Port),
-		Handler: r,
+	server := &http.Server{
+		Addr:              fmt.Sprintf(":%d", serverConf.Port),
+		Handler:           r,
+		ReadHeaderTimeout: 10 * time.Second, // Prevent Slowloris attacks
```

Comment:
Could you make this change in another pull request, so we can properly track multiple changes